doc/nrf/protocols/thread/overview/supported_features.rst

```diff
@@ -47,19 +47,21 @@ Thread 1.2 improves network scalability, responsiveness, density, and power cons
 For more information about this Thread version, see the official `Thread 1.2 in Commercial White Paper`_ and the `Thread 1.2 Base Features`_ document.
 
 .. note::
-    See :ref:`thread_ug_thread_specification_options` for the list of 1.2 features that are currently available in |NCS|.
+    See :ref:`thread_ug_thread_specification_options` for how to enable the 1.2 features that are currently available in the |NCS|.
 
 .. _thread_ug_supported_features_csl:
 
 Coordinated Sampled Listening (CSL)
 ===================================
 
-Coordinated Sampled Listening defined in IEEE 802.15.4-2015 is introduced by Thread 1.2 Specification to provide low latency communication for Sleepy End Devices.
-Thread 1.2 routers are required to support synchronized CSL transmissions to children which require them, known as Synchronized Sleepy End Devices (SSEDs).
+Coordinated Sampled Listening defined in IEEE 802.15.4-2015 is introduced by Thread 1.2 Specification to provide low latency communication for :ref:`Sleepy End Devices (SEDs)<thread_ot_device_types>`.
+Thread 1.2 routers are required to support synchronized CSL transmissions to children which require them.
+These children are known as :ref:`Synchronized Sleepy End Devices (SSEDs)<thread_ot_device_types>`.
+
 SSEDs are allowed to transmit frames normally at any time, but the routers should use the CSL transmission mechanism as long as the synchronization is maintained.
 This allows an SSED to stay in a sleepy state more than 99% of the time and only turn on its radio periodically for a few hundreds of microseconds in order to receive frames from its parent.
 
-The most common use case for SSEDs is for developing low latency battery-powered actuators, such as window blinds.
+The most common use case for SSEDs is for developing battery-powered actuators low latency, such as window blinds.
```

Comment:
Hm, can you "develop ... low latency"?

Reply:
I think I'm missing a "with" from there. Fix incoming.